src/modules/blocks/tickets/header-image/style.pcss

```
@@ -19,3 +19,59 @@
 		opacity: 0.5;
 	}
 }
+
+.tribe-editor__image-upload {
+	.tribe-editor__image-upload__content {
+		align-items: start;
+		display: flex;
+		flex-direction: column;
+		justify-content: space-between;
+	}
+
+	p.tribe-editor__image-upload__description {
+		color: var(--tec-color-icon-primary-alt);
+		font-weight: var(--tec-font-weight-regular);
+		width: 100%;
+	}
+
+	.tribe-editor__image-upload__image-wrapper {
+		padding-left: 0;
+	}
+
+	.tribe-editor__image-upload__upload-button {
+		background-color: var(--tec-color-button-primary);
+		background-image: none;
+		border-radius: var(--tec-border-radius-default);
+		border: 0;
```

I know I'm being picky, but these are out of order. 
```suggestion
		border: 0;
		border-radius: var(--tec-border-radius-default);
```

---

I have applied that suggestion. I'm now using the Visual Studio Code to sort alphabetically the properties and I didn't realize this one, Thanks @codingmusician !